### PR TITLE
Pin deps 01 3.8

### PR DESCRIPTION
**Pull request by deads2k**
`_output/depcheck pin --glide.yaml=glide.yaml --glide.lock=glide.lock` and pinning openshift/api and openshift/client-go to a branch

/assign liggitt
/assign juanvallejo

### CODE REVIEW

**Comment by deads2k** [comment]
/hold

@liggitt I don't care about whether this is before or after the rebase.

**Comment by deads2k** [comment]
@liggitt reran this

**Comment by deads2k** [comment]
/retest

They seem unrelated.  @liggitt agree that a glide.yaml update isn't likely to break conformance?

File: glide.yaml [hunk 1]
```diff
@@ -101,3 +101,388 @@ import:
   version: c9de0b92586aaba092c2a231cb7384354b178544
 - package: github.com/lestrrat/go-jspointer
   version: f4881e611bdbe9fb413a7780721ef8400a1f2341
+
```
Comment (liggitt) [inline glide.yaml]: can remove the `# pinned` ones above... that was stopgap to get a minimal diff on the rebase

Comment (deads2k) [reply glide.yaml]: > can remove the # pinned ones above and regenerate... those were stopgap to get a minimal diff on the rebase

done

**Comment by deads2k** [comment]
tested green since the rebase. nothing else has merged.  merging this to pin everything

**Pull request merged**